contentcuration/contentcuration/templates/registration/channel_published_email.txt

```diff
@@ -3,7 +3,11 @@
 {% autoescape off %}
 {% blocktrans with name=user.first_name %}Hello {{ name }},{% endblocktrans %}
 
-{% blocktrans with channel_name=channel.name %}{{ channel_name }} has finished publishing! Here is the channel token (for importing it into Kolibri):{% endblocktrans %}
+{% blocktrans with channel_name=channel.name %}{{ channel_name }} has finished publishing!{% endblocktrans %}
```

Comment:
Could maybe just do this as an `<a>` tag on the `channel_name` instead? Would mean we could release this change without additional string translation!

Reply:
> This could be slightly improved by adding the full text of the hyperlink into the email as well, allowing users to either copy paste the link, or to open it in email clients that support links.

From the issue https://github.com/learningequality/studio/issues/3560, it mentions that we paste the full hyperlink text.. thoughts?

Reply:
Hrm, that @rtibbles guy does have a point about showing the full text of the link. If we could add the link without updating the translation strings for now, that might be good so that we can release sooner rather than later, and then we can add a follow up issue to revise the strings more.

Reply:
In addition to adding `<a>` tag on channel name, we can keep the full hyperlink text between brackets just after the channel name like `<channel_name> (https://studio.learningequality.org/channels/asd56456)`. What do you think sir?

This way we won't need to update translation strings and we'll be able to address the concerns mentioned in #3560.

Reply:
Sounds very workable!

Reply:
Will do on monday sir! Currently out from office for a short diwali break sir.

Reply:
Fixed sir!